src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle(Color color, int firstSide, int secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    public double getArea() {
+        return firstSide * secondSide;
+    }
+
+    public String draw() {
```

Comment:
The method name 'draw' implies an action and might be better suited for an interface that specifies drawable behavior. Ensure that the class structure follows the single responsibility principle.